### PR TITLE
Change gql queries to template strings

### PR DESCRIPTION
**Pull request by jimbo**
## This PR is a:

- [ ] New feature
- [ ] Enhancement/Optimization
- [x] Refactor
- [x] Bugfix
- [ ] Test for existing code
- [ ] Documentation

## Summary

When this pull request is merged, it will change GraphQL queries from bare `.graphql` files to `.js` modules that export tagged template strings.

## Additional information

This should circumvent caching bugs with the query loader.

# This breaks everything because Upward expects bare queries.

@zetlen  Input on how to move forward?

### CODE REVIEW

File: packages/venia-concept/venia-upward.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ urlResolver: urlResolverResult.data.urlResolver
 
 urlResolverResult:
   url: magentoGQL
-  query: './src/queries/urlResolver.graphql'
+  query: './src/queries/urlResolver.js'
```
Comment (jimbo) [inline packages/venia-concept/venia-upward.yml]: This doesn't work.

**Comment by pcvonz** [comment]
Be sure to close this issue too if this gets merged: https://github.com/magento-research/pwa-studio/issues/552

**Comment by jimbo** [comment]
Closing in favor of #601.

**Comment by jimbo** [comment]
Closing in favor of #601.

**Pull request closed without merging**